### PR TITLE
MIFOSX-2002

### PR DESCRIPTION
**Pull request by keshav10**
MIFOSX-2002
allow search client by mobile_no
now it also working for global search also

### CODE REVIEW

**Comment by vishwasbabu** [comment]
@keshav10 : https://mifosforge.jira.com/browse/MIFOSX-2002 has already been closed. Could you please explain the change you are trying to make with this pull request ?

**Comment by keshav10** [comment]
when i sent the pull request first time for mifosx-2002 it does not search
client by through  globel search now i fixed it and it will work for global
search also

On Mon, May 18, 2015 at 11:08 AM, Vishwas Babu A J <notifications@github.com

File: mifosng-provider/src/main/java/org/mifosplatform/portfolio/search/service/SearchReadPlatformServiceImpl.java [hunk 1]
```diff
@@ -63,8 +63,10 @@ public SearchReadPlatformServiceImpl(final PlatformSecurityContext context, fina
         final MapSqlParameterSource params = new MapSqlParameterSource();
         params.addValue("hierarchy", hierarchy + "%");
         params.addValue("search", searchConditions.getSearchQuery());
-        params.addValue("partialSearch", "%" + searchConditions.getSearchQuery() + "%");
+        params.addValue("partialSearch", "%" + searchConditions.getSearchQuery()+"%");
 
+        String sql = rm.searchSchema(searchConditions);
```
Comment (vishwasbabu) [inline mifosng-provider/src/main/java/org/mifosplatform/portfolio/search/service/SearchReadPlatformServiceImpl.java]: what does this line do ? the sql string is not used anywhere

**Comment by vishwasbabu** [comment]
@keshav10 : I am not sure I understand. The pull request does not seem to have any change in it (other than formatting changes).
Can you please add a comment under any change which you have made so I can understand the change that had been made here

**Comment by vishwasbabu** [comment]
Here is the difference without whitespace diffs https://github.com/openMF/mifosx/pull/1383/files?w=1

**Comment by keshav10** [comment]
in groupExactMatchsql query i had added this join condation
join m_client c on c.office_id=g.office_id;
and in previous office hierarchy will not working in this query query it will be working

**Comment by vishwasbabu** [comment]
closing off this pull request

**Pull request closed without merging**